apps/condo/domains/contact/components/ContactsEditor/NotResidentFields.tsx

```diff
@@ -3,11 +3,14 @@ import { AutoComplete, Col, Form, Row, RowProps } from 'antd'
 import { OptionProps } from 'antd/lib/mentions'
 import { get } from 'lodash'
 import debounce from 'lodash/debounce'
+import isString from 'lodash/isString'
```

Comment:
cmon, dude :)
Sometimes I think we should add `isEven` and `isOdd` npm packages after lines like this 😄

Reply:
It's easier to understand `isString(someValue)` then `typeof someValue === 'string'` :)

Reply:
For me second one is the way to go, since I know what exactly is happening. Lodash functions can be tricky sometimes despite of simple names and do not what you exactly expect from its name.

Like this one:
<img width="992" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/fe92927d-709d-443c-980f-630db75b9ae3">

Did you expect isObject(new String('')) to be true? Many of our developers using it to check `Record<string, unknown>` with it, but according to the code none of them knows that it checks js-objects and not like datatype-objects

Reply:
> Did you expect isObject(new String('')) to be true?

Yes, because I read the documentation😄